src/pages/friend.js

```diff
@@ -17,7 +17,7 @@ import NavBar from "../components/navBar";
 import { GET } from "../utils/fetch";
 import { useState } from "react";
 
-const current_user_id = localStorage.getItem("current");
+const current_user_id = 1; //localStorage.getItem("current");
```

Comment:
Was this change intentional?  Will `current_user_id` always be 1?

Reply:
That wasn't suppposed to be that way. I dont know why but this PR has changes in files it shouldn't have. I'm only working in `profile.js` in this one (and related components `list.js` and `Info`
The up to date changes in `friend.js` are in main, I just checked. I need to solve conflicts apparently